fixed-requirements.txt

```diff
@@ -10,6 +10,7 @@ cryptography==2.8
 eventlet==0.25.1
 flex==6.14.0
 gitpython==2.1.15
+git+https://github.com/StackStorm/st2-rbac-backend.git@master#egg=st2-rbac-backend
```

Comment:
```suggestion
```
as the dependency is in `st2common/in-requirements.txt`

Reply:
Updated global requirements